index.html

```diff
@@ -0,0 +1,203 @@
+                  <img class="speaker-images card2-img" src="./sources/speaker2.jpg" alt="Card2-image">
+                  <img class="speaker-bgimages" src="./sources/grey checker.png" alt="Card1-bgimage">
+                  <div class="card-body">
+                    <h5 class="speaker-name">Aurora</h5>
+                    <p class="speaker-title">3D Illustrator</p>
+                    <p class="speaker-bio" alt="speaker-bio">This is a longer card with supporting text. </p>
+                  </div>
+                </div>
+    
+                <div class="each-card" id="speakerCard3">
+                  <img class="speaker-images card3-img" src="./sources/speaker3.jpg" alt="Card3-image">
+                  <img class="speaker-bgimages" src="./sources/grey checker.png" alt="Card1-bgimage">
+                  <div class="card-body">
+                    <h5 class="speaker-name">Blanca</h5>
+                    <p class="speaker-title">Graphics Designer</p>
+                    <p class="speaker-bio" alt="speaker-bio">This is a longer card with supporting text.</p>
+                  </div>
+                </div>
+
+             
+  
+           
+                <div class="each-card" id="speakerCard4">
+                  <img class="speaker-images card4-img" src="./sources/speaker4.jpg" alt="Card4-image">
+                  <img class="speaker-bgimages" src="./sources/grey checker.png" alt="Card1-bgimage">
+                  <div class="card-body">
+                    <h5 class="speaker-name">Gibron</h5>
+                    <p class="speaker-title">Voice Editor</p>
+                    <p class="speaker-bio" alt="speaker-bio">This is a longer card with supporting text.</p>
+                  </div>
+                </div>
+    
+                <div class="each-card" id="speakerCard5">
+                  <img class="speaker-images card5-img" src="./sources/speaker5.jpg" alt="Card5-image">
+                  <img class="speaker-bgimages" src="./sources/grey checker.png" alt="Card1-bgimage">
+                  <div class="card-body">
+                    <h5 class="speaker-name">Keller</h5>
+                    <p class="speaker-title">AfterEffects Artist</p>
+                    <p class="speaker-bio" alt="speaker-bio">This is a longer card with supporting text.</p>
+                  </div> 
+                </div>
+    
+                <div class="each-card" id="speaker-card6">
+                  <img class="speaker-images card6-img" src="./sources/speaker6.jpg" alt="Card6-image">
+                  <img class="speaker-bgimages" src="./sources/grey checker.png" alt="Card1-bgimage">
+                  <div class="card-body">
+                    <h5 class="speaker-name">Joy</h5>
+                    <p class="speaker-title">Video Editor</p>
+                    <p class="speaker-bio" alt="speaker-bio">This is a longer card with supporting text.</p>
+                  </div>
+                </div>-->
+            
```

Comment:
- [ ] Good job working on the speaker's section 👍.Everything is correct in your code. but to make it more professional, you should remove these line comments, I mean `110` to `174`. Kindly, fix it 👍